Foundation/NSString.swift

```diff
@@ -331,7 +354,8 @@ public class NSString : NSObject, NSCopying, NSMutableCopying, NSSecureCoding, N
     }
     
     public override var hash: Int {
-        return Int(CFStringHashNSString(self._cfObject))
+        return unsafeBitCast(CFStringHashNSString(self._cfObject), Int.self)
```

Comment:
I think the underlying problem here is that hash is Int and CF/Foundation's hash ought to be UInt; you probably want Int(bitPattern:CFStringHashNSString(self._cfObject))